### DISCUSSION TITLE
桌面版是否支持反向激活？

### DISCUSSION THREAD

**Discussion by free-150** [AlistGo/alist#5287] [Q&A] [answered]
现在主流的激活有两种。一种是帐号那种，超出设备限制的最大数时最早登录的设备会掉激活。另一种是在原先激活的设备上设备上点击「反激活」这样这台设备就会失效空出一台设备在新设备上激活。不知道 alist 桌面版是否支持这其中一种方式。

**Comment by xhofe** [comment, marked as answer]
支持在原设备上或https://ad.nn.ci/usage 中点击「反激活」
